v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -1,100 +1,105 @@
 import React, {useCallback, useEffect, useRef, useState} from "react"
 import {drag, ScaleLinear, select} from "d3"
-import {CPLine, equationString, lineToAxisIntercepts} from "../utilities/graph_utils"
+import {equationString, lineToAxisIntercepts} from "../utilities/graph_utils"
 import "./movable-line.scss"
+import {IMovableLineModel} from "./adornment-models"
+import {autorun} from "mobx"
```

Comment:
Conventional import ordering would have the `mobx` import in the top section with the other third-party modules and the `.scss` would be the last import.